### PR TITLE
Add detail post for upcoming conferences

### PR DESCRIPTION
**Pull request by GregSutcliffe**

### CODE REVIEW

File: _posts/2016-01-11-upcoming-conferences-and-events-in-the-foreman-community.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+- contributing
+---
+
+With FOSDEM, ConfigManagement Camp, DevConf, and our first ever Foreman
+Construsction Day fast approaching, I thought you might like a bit more detail
+on our plans. Read on if you do!
+
+<!--more-->
+
+## [FOSDEM 2016](fosdem.org), 30-31 January, Brussels
```
Comment (domcleal) [inline _posts/2016-01-11-upcoming-conferences-and-events-in-the-foreman-community.md]: Links must be full URLs, use http://

File: _posts/2016-01-11-upcoming-conferences-and-events-in-the-foreman-community.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+For talks, check out the Config Management, Virtualization, and Container
+devrooms for things that are relevant to Foreman.
+
+There's also going to be a Foreman Community dinner on the Saturday night
+(30th). Venue is still to be confirmed, but if you want to join us, the latest
+are on the [Google+ event](https://plus.google.com/b/102496134326414788199/events/co1nia0c7g6hk9b4msf98517s1o).
+Sign up there, or mail me directly if you're not a fan of G+.
+
+## [ConfigManagement Camp 2016](cfgmgmtcamp.eu), 1-2 February, Ghent
```
Comment (domcleal) [inline _posts/2016-01-11-upcoming-conferences-and-events-in-the-foreman-community.md]: ditto

File: _posts/2016-01-11-upcoming-conferences-and-events-in-the-foreman-community.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+[published](http://cfgmgmtcamp.eu/schedule/index.html#foreman) and looks
+fantastic! If you don't already have your ticket, get on the
+[waiting list](http://cfgmgmtcamp.eu/#registration) now, and hopefully we can
+squeeze you in.
+
+Socially, there's a general drinkup event on Monday for the whole conference,
+and no doubt something impromptu will happen on Tuesday night. Which leads us
+to...
+
+## [Foreman Construction Day](www.eventbrite.com/e/foreman-construction-day-registration-19911909056), 3 February, Ghent
```
Comment (domcleal) [inline _posts/2016-01-11-upcoming-conferences-and-events-in-the-foreman-community.md]: ditto, missing protocol

File: _posts/2016-01-11-upcoming-conferences-and-events-in-the-foreman-community.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+[Eventbrite page](www.eventbrite.com/e/foreman-construction-day-registration-19911909056).
+
+Since we're in the same venue, the same catering applies as for
+ConfigManagement - we'll be getting breakfast, lunch, and coffee/cake during
+breaks. We should have a couple of rooms, so there's space to spread out and
+get things done.
+
+If you're interested in being part of this, and aren't signed up yet, please
+grab a (free!) ticket at the event link above - tickets mean we can get the
+caterng numbers right :)
```
Comment (domcleal) [inline _posts/2016-01-11-upcoming-conferences-and-events-in-the-foreman-community.md]: catering

**Comment by GregSutcliffe** [comment]
Thanks @domcleal all fixed, another typo, and added a couple more links.

**Comment by domcleal** [comment]
Thanks @GregSutcliffe.

**Pull request merged**